data.csv

```diff
@@ -14,7 +14,7 @@ Square,16,16,16,,kim@disqus.com,,,
 Palantir,16,4,,,kim@disqus.com,,,
 Jawbone,16,8,8,,kim@disqus.com,,,
 Github,16,16,,,kim@disqus.com,,,
-Disqus,12,8,"12 weeks primary caregiver; 8 weeks secondary caregiver - not maternal/paternal. both leaves coordinate with FMLA, PDL, CFRA, etc",,kim@disqus.com,,,
+Disqus,12,8,"12 weeks primary caregiver;,8 weeks secondary caregiver - not maternal/paternal. both leaves coordinate with FMLA, PDL, CFRA, etc",kim@disqus.com,,,
```

Comment:
Hi @kimskitchensink I think you are missing a comma after the "8," before the `"`.

You can probably edit this again?